src/components/20-molecules/image-upload/index.scss

```diff
@@ -0,0 +1,179 @@
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    align-items: flex-start;
+    overflow-y: scroll;
+    border-style: solid;
+  }
+
+  &__or {
+    text-transform: uppercase;
+    font-weight: bold;
+    color: $color-prim-red-flamingo;
+    margin: 0;
+  }
+
+  &__information {
+    margin: 0;
+  }
+
+  &__input {
+    fill: currentColor;
+  }
+
+  &__img-figure {
+    position: relative;
+    margin: 0;
+    margin-right: 15px;
+    margin-bottom: 25px;
+  }
+
+  &__icon-hover-area {
+    position: relative;
+    height: 80px;
+    width: 120px;
+  }
+
+  &__icon-hover-area:hover {
+    cursor: pointer;
+    .m-image-upload__img-element {
+      opacity: $image-upload-opacity;
+    }
+    .m-image-upload__file-element {
+      opacity: $image-upload-opacity;
+    }
+    .m-image-upload__icon-delete {
+      display: block;
+    }
+    .m-image-upload__icon-error {
+      display: none;
+    }
+  }
+
+  &__img-element {
+    width: 100%;
+    height: 100%;
+  }
+
+  &__file-element {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%);
+    // height: 36px; // TODO
+  }
+
+  &__icon-layer {
+    position: absolute;
+    height: 36px;
+    top: 50%;
+    left: 50%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%);
+  }
+
+  &__icon-delete {
+    display: none;
+    fill: $color-axa-blue;// TODO needed?
+    color: $color-axa-blue;
+  }
+
+  &__icon-error {
+    fill: $color-sec-shy-tomato;// TODO needed?
+    color: $color-sec-shy-tomato;
+  }
+
+  &__img-caption {
+    position: relative; // TODO
+    width: 120px;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    text-align: center;
+    cursor: default;
+  }
+
+  &__img-caption-error {
+    color: $color-sec-shy-tomato;
+  }
+
+  // when hovering over hover-area, hide the filname
+  &__icon-hover-area:hover ~ .m-image-upload__img-caption {
+    .m-image-upload__filename {
+      display: none;
+    }
+  }
+
+  // show delete-status instead of filename
+  &__icon-hover-area:hover ~ .m-image-upload__img-caption[data-status]:after {
+      content: attr(data-status);
+  }
+
+  // color delete-status and ellipsis
+  &__icon-hover-area:hover ~ .m-image-upload__img-caption[data-status] {
+    color: $color-axa-blue;
+  }
+
+  &__add-more {
+    position: relative;
+    height: 80px;
+    width: 120px;
+    box-sizing: border-box;
+    cursor: pointer;
+    border: $image-upload-dropzone-border;
+    color: $color-prim-gray-dusty;
+    .m-image-upload__icon-wrapper {
+      height: 100%;
+      width: 100%;
+      display: block;
+      position: relative;
+    }
+    .m-image-upload__icon-layer {
+      fill: currentColor;
+    }
+    .m-image-upload__img-caption {
+      width: 116px;
+      margin-top: 2px;
+      position: block;
+    }
+
+  }
+
+  &__error-wrapper {
+    color: $color-validation-error;
+  }
+
+  &__error-wrapper--reservation {
```

Comment:
With newest design change we don't need anymore so reserve space for the error message.

Reply:
Oke I removed it from the code & css